tests/evaluate/test_default_evaluator.py

```diff
@@ -4217,6 +4223,30 @@ def test_log_genai_custom_metrics_as_artifacts():
     assert table.loc[1, "version"] == ""
     assert table["version"].dtype == "object"
 
+    results = retrieve_custom_metrics(run.info.run_id)
+    assert len(results) == 2
```

Comment:
```suggestion
```

I'd remove this line because the line below validates that it has two items, but fine to keep it.

Reply:
yah this one is more of a personal preference as I want the test fail with more targeted error message, instead of testing length and names in the same assert.